lib/rucio/daemons/reaper/reaper2.py

```diff
@@ -436,6 +444,9 @@ def reaper(rses, include_rses, exclude_rses, vos=None, chunk_size=100, once=Fals
             GRACEFUL_STOP.wait(30)
             continue
         start_time = time.time()
+        memcached_expiration_time = get('reaper', 'memcached_expiration_time')
```

Comment:
Let's call this `pause_deletion_time` or so. It has nothing todo with memcache per se, that's just the mechanism how it is implemented.